packages/create-plugin/src/commands/update.command.ts

```diff
@@ -1,81 +1,80 @@
+      });
 
-    // 1. Add / update configuration files
-    // -----------------------------------
-    if (await confirmPrompt(TEXT.updateConfigPrompt)) {
-      compileTemplateFiles(UDPATE_CONFIG.filesToOverride, getTemplateData());
-      printSuccessMessage(TEXT.overrideFilesSuccess);
-    } else {
-      printMessage(TEXT.overrideFilesAborted);
-      process.exit(0);
+      process.exit(1);
     }
 
-    // 2. Add / update dev dependencies inside the `package.json`
-    // (skipped automatically if there is nothing to update)
-    // ------------------------------------------------
-    if (hasNpmDependenciesToUpdate({ devOnly: true })) {
-      const updatableText = getPackageJsonUpdatesAsText({ ignoreGrafanaDependencies: true, devOnly: true });
+    if (!(await isGitDirectoryClean())) {
+      printRedBox({
+        title: 'Please clean your repository working tree before updating.',
+        subtitle: '(Commit your changes or stash them.)',
+        content: `(This check is necessary to make sure that the updates are easy to revert and don't mess with any changes you currently have.
+In case you want to proceed as is please use the ${chalk.bold('--force')} flag.)`,
+      });
 
-      if (updatableText.length > 0) {
-        const PROMPT_CHOICES = {
-          ALL: 'Yes, all of them',
-          ONLY_OUTDATED: 'Yes, but only the outdated ones',
-          NONE: 'No',
-        };
+      process.exit(1);
+    }
 
-        const shouldUpdateDeps = await selectPrompt(TEXT.updateNpmDependenciesPrompt + updatableText, [
-          PROMPT_CHOICES.ALL,
-          PROMPT_CHOICES.ONLY_OUTDATED,
-          PROMPT_CHOICES.NONE,
-        ]);
+    if (!isPluginDirectory()) {
+      printRedBox({
+        title: 'Are you inside a plugin directory?',
+        subtitle: 'We couldn\'t find a "src/plugin.json" file under your current directory.',
+        content: `(Please make sure to run this command from the root of your plugin folder. In case you want to proceed as is please use the ${chalk.bold(
+          '--force'
+        )} flag.)`,
+      });
 
-        if (shouldUpdateDeps && shouldUpdateDeps !== PROMPT_CHOICES.NONE) {
-          updatePackageJson({
-            onlyOutdated: shouldUpdateDeps === PROMPT_CHOICES.ONLY_OUTDATED,
-            ignoreGrafanaDependencies: true,
-            devOnly: true,
-          });
-          printSuccessMessage(TEXT.updateNpmDependenciesSuccess);
-        } else {
-          printMessage(TEXT.updateNpmDependenciesAborted);
-        }
-      }
+      process.exit(1);
     }
 
-    // 3. Add / update NPM scripts
-    // (skipped automatically if there is nothing to update)
-    // ------------------------------------------------
-    if (await confirmPrompt(TEXT.updateNpmScriptsPrompt)) {
-      updateNpmScripts();
-      printSuccessMessage(TEXT.updateNpmScriptsSuccess);
-    } else {
-      printMessage(TEXT.updateNpmScriptsAborted);
-    }
+    updatePluginBuildConfig();
```

Comment:
A better name for this is `updateDotConfigFolder` I was a bit confused what was the "build config"